### PR TITLE
Closes #3035: Add 'Configure' Links to Quickstart Modules

### LINKED ISSUE #3035: Feature Request: Add 'Configure' Links to Quickstart Modules
### Problem:
Currently, administrators may encounter difficulties in locating the configuration sections for various Quickstart modules, leading to inefficiencies.

### Proposed Solution:
See https://www.drupal.org/docs/develop/creating-modules/let-drupal-know-about-your-module-with-an-infoyml-file#s-complete-example

Add a "Configure" link to the following Quickstart modules, enabling administrators to easily access and manage their settings:

1. **AZ Publication**
   - Path: `/admin/config/az-quickstart/settings/az-publication`
   - Purpose: Streamline content management in AZ Publication.
image

2. **AZ Core**
   - Path: `/admin/config/az-quickstart/settings`
   - Purpose: Essential for easy access to the core settings of Quickstart.
image

3. **AZ CAS**
   - Path: `/admin/config/az-quickstart/settings/az-cas`
   - Purpose: Facilitate quicker adjustments to CAS settings.
image

~~4. **AZ Event Trellis**~~
   ~~- Path: `/admin/config/az-quickstart/settings/az-event-trellis`~~
   ~~- Purpose: Aid in efficiently managing event configurations.~~
image

~~5. **AZ Flexible Page**~~
   ~~- Path: `/admin/config/az-quickstart/settings/az-flexible-page`~~
   ~~- Purpose: Improve the ease of modifying flexible page settings.~~
image

6. **AZ Enterprise Attributes Import**
   - Path: `/admin/config/az-enterprise-attributes-import/settings`
   - Purpose: Ensure easier management of enterprise attributes import.
image

### Benefits:
- **Enhanced Accessibility:** Quick and direct access to module settings.
- **Increased Efficiency:** Reduces time spent searching for configuration options.
- **Improved Usability:** Streamlines module management for administrators.

Adding 'Configure' links to these modules will greatly enhance the usability and efficiency of the Quickstart environment for administrators.

### PR DESCRIPTION
**Pull request by maine-inventor**
## Related issues

Closes #3035

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [x] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_core/az_core.info.yml [hunk 1]
```diff
@@ -16,3 +16,4 @@ dependencies:
   - drupal:text
   - linkit:linkit
   - search_exclude:search_exclude
+configure: az-core.settings
```
Comment (trackleft) [inline modules/custom/az_core/az_core.info.yml:19]:
```suggestion
configure: az_core.settings
```

File: modules/custom/az_cas/az_cas.info.yml [hunk 2]
```diff
@@ -6,3 +6,4 @@ package: 'The University of Arizona'
 dependencies:
   - cas:cas
   - az_core
+configure: az-cas.settings
```
Comment (trackleft) [inline modules/custom/az_cas/az_cas.info.yml:9]:
```suggestion
configure: az_cas.settings
```

Comment (trackleft) [reply modules/custom/az_cas/az_cas.info.yml:9]: See https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_cas/az_cas.routing.yml#L1

File: modules/custom/az_core/az_core.info.yml (hunk 1 above)
Comment (trackleft) [reply modules/custom/az_core/az_core.info.yml:19]: https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_core/az_core.routing.yml#L9

File: modules/custom/az_core/az_core.info.yml [hunk 3]
```diff
@@ -16,3 +16,4 @@ dependencies:
   - drupal:text
   - linkit:linkit
   - search_exclude:search_exclude
+configure: az_core.settings
```
Comment (trackleft) [inline modules/custom/az_core/az_core.info.yml:19]:
```suggestion
configure: az_core.az_settings
```

**Review by joeparsons — APPROVED** [review]

**Comment by trackleft** [comment]
image

**Review by trackleft — APPROVED** [review]

**Comment by trackleft** [comment]
image

**Review by maine-inventor — COMMENTED** [review]
Completed.

**Review by tadean — APPROVED** [review]

**Review by danahertzberg — APPROVED** [review]

**Pull request merged**